docs/source/contribute.rst

```diff
@@ -35,7 +35,7 @@ forward on all code in PyU4V.
 
 2. Use ``.format()`` for string concatenation.
 
-.. code-block::
+.. code-block:: text
```

Comment:
Code block should be Python

Reply:
Again it complained on the make html.  I will take another look